wagtailflags/views.py

```diff
@@ -42,6 +43,28 @@ def create_flag(request):
     return render(request, "wagtailflags/flags/create_flag.html", context)
 
 
+def delete_flag(request, name):
+    """ Delete a database flag. """
+    flag = get_flags().get(name)
+
+    if not flag:
+        raise Http404
+
+    if not deletable(flag):
+        return HttpResponseNotAllowed(request.method)
```

Comment:
Is [HTTP 405 Method Not Allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) really the right code here? It's not the request method that's incorrect (POST vs something else), it's that the flag isn't deletable. Should this be [HTTP 403 Forbidden](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403) to be more consistent with how Wagtail handles permission-related errors?

Reply:
Done in 3f741b7.